demo-shell/src/app/components/files/files.component.ts

```diff
@@ -591,6 +596,27 @@ export class FilesComponent implements OnInit, OnChanges, OnDestroy {
         this.logService.log(event);
     }
 
+    onUploadNewVersion(ev) {
```

Comment:
does this mean every ADF application will have to have this code implemented on the app level?

Reply:
yes it does